AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/AWSModelReconciliationQueue.swift

```diff
@@ -52,6 +52,11 @@ final class AWSModelReconciliationQueue: ModelReconciliationQueue {
 
     weak var storageAdapter: StorageEngineAdapter?
 
+    internal var isFullSync: Bool
+    internal var count: Int
+    var completionCount = AtomicValue.init(initialValue: 0)
```

Comment:
What is the difference between `completionCount` and the sum of each member of the `modelSyncedEventBuilder`?